torch/jit/_monkeytype_config.py

```diff
@@ -14,6 +14,25 @@
 except ImportError:
     _IS_MONKEYTYPE_INSTALLED = False
 
+def get_element_type(types: str):
+    """
+    Helper function to extract the type of the element to be annotated to Optional
```

Comment:
It is a bit odd that this `get_element_type` eventually returns an `optional` when the name implies it should return the underlying element type.

Maybe we should simply rename this function to make it clear that it may create an optional.

Reply:
Done.